iceoryx_posh/test/moduletests/test_runtime_ipc_interface.cpp

```diff
@@ -333,7 +334,7 @@ TYPED_TEST(IpcInterface_test, TimedReceiveWorks)
     ASSERT_FALSE(this->server->timedReceive(timeout, receivedMessage));
     auto after = system_clock::now();
 
-    auto timeDiff = units::Duration(after - before);
-    EXPECT_GE(timeDiff, timeout);
+    auto timeDiff = std::chrono::duration_cast<std::chrono::milliseconds>(after - before);
+    EXPECT_GE(into<units::Duration>(timeDiff), timeout);
```

Comment:
@FerdinandSpitzschnueffler For Windows and MSVC an explicit `duration_cast` is needed otherwise an implementation for `into` from the type is needed `iox::into<iox::units::Duration,std::chrono::duration<std::chrono::system_clock::rep,std::chrono::system_clock::period>>`, see [here](https://github.com/eclipse-iceoryx/iceoryx/actions/runs/4374027889/jobs/7652951288).

Reply:
@FerdinandSpitzschnueffler on top of what @mossmaurice wrote, I'm not even fully sure why it worked before. `now` returns a `time_point` and subtracting two `time_points` returns some `common_type::type` template magic which tries to deduce the correct type. I guess it just happens to work by accident.